backend/src/lib/factories/user-factory.ts

```diff
@@ -0,0 +1,25 @@
+import { User } from "../models/user";
+import { UserType } from "aws-sdk/clients/cognitoidentityserviceprovider";
+
+function toUser(cognitoUser: UserType): User {
+  return {
+    userId: cognitoUser.Username || "",
+    enabled: cognitoUser.Enabled,
+    userStatus: cognitoUser.UserStatus,
+    sub: cognitoUser.Attributes ? cognitoUser.Attributes[0].Value : "",
+    emailVerified: cognitoUser.Attributes
+      ? cognitoUser.Attributes[1].Value === "true"
```

Comment:
Is this attribute always in the position `1` of the Attributes array? I'm not sure if we should rely on that index. Also, we don't need to know if the email is verified. I would suggest you can get rid of this field.